### PR TITLE
More BuildOutput small fixes

### PR DESCRIPTION
**Pull request by rodrmoya**

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputWidget.cs [hunk 1]
```diff
@@ -122,20 +124,33 @@ void OnOutputChanged (object sender, EventArgs args)
...
+
+		void OnProjectFinished (object sender, EventArgs args)
+		{
+			SetSpinnerVisibility (false);
+		}
+
 		void Initialize (DocumentToolbar toolbar)
 		{
 			Spacing = 0;
+			var nativeToolbar = toolbar.Container.GetNativeWidget<Gtk.Widget> ();
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputWidget.cs]: What is nativeToolbar used for?

Comment (rodrmoya) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputWidget.cs]: Nothing, seems to be a left over, so will remove

Comment (rodrmoya) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.BuildOutputView/BuildOutputWidget.cs]: removed

**Comment by slluis** [comment]
@monojenkins backport release-7.6

**Pull request merged**